nautobot/docs/user-guide/index.md

```diff
@@ -1 +1,62 @@
-# User Guide
+# User Guide Overview
+
+Welcome to the Nautobot User Guide! The User Guide is designed to help you setup, use and operate your Nautobot installation. 
+
+There are four main headings within the guide: 
+
+- **Administration**: Go here for [installation](./administration/installation/index.md), [migration](./administration/migration/migrating-from-netbox.md), [configuration](./administration/configuration/index.md) and [upgrade instructions](./administration/upgrading/upgrading.md).
+- **Feature Guides**: Walk through these guides if you are a [new user to Nautobot](./feature-guides/getting-started/index.md) and wondering how perform common tasks in the Web UI. 
```

Comment:
up for debate, but we have nautobot navigators that may make sense to point to.

Reply:
Let's maybe have the fourth quick links panel for new users be links to any useful media like the navigators playlist?

Reply:
Good idea. Navigators and I think 100 days of nautobot is also a good thing to point to IMHO